src/main/java/frc/robot/RobotContainer.java

```diff
@@ -228,19 +228,30 @@ private void configureButtonBindings() {
       Back.whenPressed(() -> m_robotDrive.makeBackwards(true));
       Start.whenPressed(() -> m_robotDrive.makeBackwards(false));
 
-      AButton.whenHeld(new RunCommand(() -> {m_robotDrive.driveNoDeadband(0, 0, m_limelight.getRotationSpeed());
-      m_limelight.ledsOn();}, m_robotDrive));
-      BButton.whenPressed(new InstantCommand(() -> m_robotDrive.resetOdometry(new Pose2d(-1.06+0.444, -2.76+0.444, new Rotation2d(0,-1)))));
+      // AButton.whenHeld(new RunCommand(() -> {m_robotDrive.driveNoDeadband(0, 0, m_limelight.getRotationSpeed());
+      // m_limelight.ledsOn();}, m_robotDrive));
+      AButton.whenHeld(new SequentialCommandGroup(new RunCommand(()-> {m_robotDrive.driveNoDeadband(0, 0, m_limelight.getRotationSpeed());
```

Comment:
Why not spin up the launcher while aiming?

Reply:
Fair, that is a good idea, I'll do that.